### ISSUE TITLE
求conditional batch normalization

### ISSUE THREAD

**Issue by biubiuyi** [PaddlePaddle/Paddle#28256]
想要实现conditional batch normalization，请问有没有静态图的实现参考呢，在源码中只看到BN

**Comment by chenxiaozeng** [comment]
您好，您的问题已反馈给相关研发同学，请耐心等待下~

**Comment by biubiuyi** [comment]
> 您好，您的问题已反馈给相关研发同学，请耐心等待下~

好的，期待回复～

**Comment by qingqing01** [comment]
@biubiuyi 


看样子可以参考

https://github.com/crcrpar/pytorch.sngan_projection/blob/master/links/conditional_batchnorm.py#L6

结合functional的batch_norm:

https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/functional/norm.py#L117

**Comment by biubiuyi** [comment]
> @biubiuyi
> 
> 看样子可以参考
> 
> https://github.com/crcrpar/pytorch.sngan_projection/blob/master/links/conditional_batchnorm.py#L6
> 
> 结合functional的batch_norm:
> 
> https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/functional/norm.py#L117

感谢，我参考下

**Comment by qingqing01** [comment]
@biubiuyi 忘记提醒了，functional的BatchNorm在2.0rc版本才有。
